packages/framework-provider-azure-infrastructure/src/test-helper/azure-counters.ts

```diff
@@ -10,13 +17,16 @@ export class AzureCounters {
     return 0
   }
 
-  //TODO we should implement this for provider unaware functionality tests
   public async events(): Promise<number> {
-    return 0
+    return await this.itemsCount(`${this.appName}-app-events-store`, 'SELECT * FROM c WHERE c.kind="event"')
```

Comment:
What do you think about using `count` here in line 25? Perhaps we could do those queries faster like this:


```suggestion
    return await this.itemsCount(`${this.appName}-app-events-store`, 'SELECT COUNT(1) as total FROM c WHERE c.kind="event"')
```

And in the `itemsCount` method returns the `total` value instead of the length?

Reply:
changed 🎩